src/indra_cogex/apps/queries_web/__init__.py

```diff
@@ -106,14 +279,39 @@ def get_example_data():
     # Example for /gene/discrete
     "gene_list": fields.List(fields.String, example=EXAMPLE_GENE_IDS),
     # Examples for positive_genes and negative_genes for /gene/signed
-    "positive_genes": fields.List(fields.String,example=EXAMPLE_POSITIVE_HGNC_IDS),
-    "negative_genes": fields.List(fields.String,example=EXAMPLE_NEGATIVE_HGNC_IDS),
+    "positive_genes": fields.List(fields.String, example=EXAMPLE_POSITIVE_HGNC_IDS),
+    "negative_genes": fields.List(fields.String, example=EXAMPLE_NEGATIVE_HGNC_IDS),
     "gene_names": fields.List(fields.String, example=continuous_analysis_example_names),
     "log_fold_change": fields.List(fields.Float, example=continuous_analysis_example_data),
     "species": fields.String(example="human"),
     "permutations": fields.Integer(example=100),
     "source": fields.String(example="go"),
     "indra_path_analysis": fields.Boolean(example=False),
+    # Cell marker
+    "cell_type": fields.List(fields.String, example=["cl", "0000020"]),
+    "marker": fields.List(fields.String, example=["hgnc", "11337"]),
+    # Pubmed
+    "publication": fields.List(fields.String, example=["pubmed", "14334679"]),
+    "journal": fields.List(fields.String, example=["nlm", "0000201"]),
+    # Disgenet
+    "variant": fields.List(fields.String, example=[["dbsnp", "rs9994441"], ["dbsnp", "rs13015548"]]),
+    # Wikidata
+    "publisher": fields.List(fields.String, example=["isni", "0000000031304729"]),
+    # NIH Reporter
+    "project": fields.List(fields.String, example=["nihreporter.project", "2106659"]),
+    "patent": fields.List(fields.String, example=["google.patent", "US5939275"]),
+    # HPOA
+    "phenotype": fields.List(fields.String, example=[["hp", "0003138"], ["mesh", "D001827"]]),
+    # For InterPro
+    "domain": fields.List(fields.String, example=["interpro", "IPR006047"]),
+    # For DepMap codependency
+    "gene1": fields.List(fields.String, example=["hgnc", "1234"]),
+    "gene2": fields.List(fields.String, example=["hgnc", "5678"]),
+    # For ChEMBL
+    "molecule": fields.List(fields.String, examples=[["chebi", "10001"], ["chembl.compound", "CHEMBL25"]]),
```

Comment:
I think this is a typo, and just like above, we need to use only one of the curies.
```suggestion
    "molecule": fields.List(fields.String, example=["chebi", "10001"]),
```

Reply:
Fixed in 4640b451